extensions/pyRevitTools.extension/pyRevit.tab/Drawing Set.panel/views.stack/Legends.pulldown/Copy Legends to Other Documents.pushbutton/script.py

```diff
@@ -40,6 +40,20 @@ def OnDuplicateTypeNamesFound(self, args):
             forms.alert('At least one Legend must exist in target document.',
                         exitscript=True)
 
+        # get reference plane category, subcategories in destination document
+        src_plane_category = \
+            DB.Category.GetCategory(
+                revit.doc, DB.BuiltInCategory.OST_CLines)
+        dest_plane_category = \
+            DB.Category.GetCategory(
+                dest_doc, DB.BuiltInCategory.OST_CLines)
+        dest_subcats = {subcat.Name:subcat 
+                        for subcat in dest_plane_category.SubCategories}
+        dest_pats = {pat.Name:pat for pat in 
+                     DB.FilteredElementCollector(dest_doc)\
```

Comment:
here you can use [get_elements_by_class](https://docs.pyrevitlabs.io/reference/pyrevit/revit/db/query/#pyrevit.revit.db.query.get_elements_by_class)